DSA/Algorithms/Asymptotic_Analysis.md

```diff
@@ -0,0 +1,103 @@
+# Asymptotic Analysis of Algorithms
+
+
+### 🤔 What is Asymptotic Analysis?
+Asymptotic Analysis is the general methodolgy for studying the performance of algorithms, especially their run time and space requirements. It is used to determine an algorithm's **efficiency**, taking into consideration the best case, average case and the worst case inputs. 
+In asymptotic notations, we derive the complexity concerning the size of the input. In simple words, it helps us determine **how the running time (or space) would be affected as the input size goes on increasing**. </br>
```

Comment:
This Line is taken from https://www.vdube.com/articles/read/discuss-asymptomatic-notations-in-brief_808.html. Kindly change the speech lil bit , otherwise it's a plagirised content.

Reply:
yes, made the necessary changes